ch.ivyteam.smart.core/src/ch/ivyteam/smart/core/tool/impl/IvyFormSchemaTool.java

```diff
@@ -35,7 +35,7 @@ public String guidelines() {
       - The form process must be of kind 'HTML_DIALOG' and contain a 'HtmlDialogStart' connected to a 'HtmlDialogEnd' and a 'HtmlDialogEventStart' connected to a 'HtmlDialogExit'.
       - The form must be located in a subdirectory of '<project-root>/""" + DIRECTORY_SRC_HD + """
       /' but not directly in it. At least one other directory between '""" + DIRECTORY_SRC_HD + """
-      ' and the form directory is required.""";
+      ' and the form directory is required. The directory in which the form is located must have the same name as the form.""";
```

Comment:
do we already validate this on the form-RDM? 
I think if we would, co-pilot could auto-heal this situation without further guidelines.